### PR TITLE
move_base_msgs: Add Feedback message

### PR DESCRIPTION
**Pull request by michaelblindnology**

### CODE REVIEW

File: move_base_msgs/msg/Feedback.msg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+float32 dist# Path distance left to goal(m)
```
Comment (pavloblindnology) [inline move_base_msgs/msg/Feedback.msg]: Design document says:
float32 dist_to_goal    # Path distance left to goal (m)
float32 time_to_goal   # Time left to goal to navigate along path (s)

File: move_base_msgs/CMakeLists.txt [hunk 2]
```diff
@@ -14,6 +14,7 @@ add_message_files(
     msg
   FILES
     RecoveryStatus.msg
+    Feedback.msg
```
Comment (pavloblindnology) [inline move_base_msgs/CMakeLists.txt:16]: Usually they are sorted alphabetically here.

Comment (michaelblindnology) [reply move_base_msgs/CMakeLists.txt:16]: No problem

File: move_base_msgs/msg/Feedback.msg (hunk 1 above)
Comment (michaelblindnology) [reply move_base_msgs/msg/Feedback.msg]: No problem

**Review by pavloblindnology — APPROVED** [review]

**Pull request merged**